Cargo.toml

```diff
@@ -31,7 +31,7 @@ doctest = false
 test = false
 
 [patch.'crates-io']
-grammer = { git = "https://github.com/lykenware/grammer", rev = "e108acbe83271761538bf5b2ff2daeaaafa5919c" }
+grammer = { git = "https://github.com/lykenware/grammer", rev = "d4f2dcd2e35c31cee0acd08ad5bcc898704004a9" }
```

Comment:
To avoid downstream users having to specify the `grammer` revision, this should be upgraded to a "proper" git dependency rather than going through `[patch.crates-io]` which only applies from the root manifest.

Reply:
I want to get rid of this soon by releasing the first version of `grammer`, once it's useful by itself.